apps/form-gateway/src/gateway/router.ts

```diff
@@ -45,15 +47,52 @@ export function verifyCaptcha(logger: Logger, RECAPTCHA_SECRET: string, SCORE_TH
   };
 }
 
+async function getSimpleFormResponse(
+  formApiUrl: URL,
+  criteria: string,
+  tenantId: AdspId,
+  tokenProvider: TokenProvider
+): Promise<SimpleFormResponse> {
+  const token = await tokenProvider.getAccessToken();
+
+  const obj = JSON.parse(criteria);
+
+  //formId is contained in the last item of the split array
+  const formId = obj.recordIdContains ? obj.recordIdContains.split('/').at(-1) : '';
+
+  const targetPath = `${formApiUrl.toString()}form/v1/forms/${formId}`;
+  try {
+    const { data } = await axios.get(targetPath, {
+      headers: { Authorization: `Bearer ${token}` },
+      params: { tenantId: tenantId.toString() },
+    });
+
+    const dataResult = data
+      ? {
+          formDefinitionId: data.definition?.id ?? null,
+          id: data.id ?? null,
+          status: data.status ?? null,
+          submitted: data.submitted ?? null,
+        }
+      : null;
+
+    return dataResult;
+  } catch (err) {
+    return null;
+  }
+}
+
 async function getFile(
   fileApiUrl: URL,
   tokenProvider: TokenProvider,
   criteria: string,
   tenantId: AdspId
 ): Promise<string> {
   const token = await tokenProvider.getAccessToken();
+
   const encodedCriteria = encodeURIComponent(criteria);
   const targetPath = fileApiUrl.toString() + '/files' + `?criteria=${encodedCriteria}` + `&tenantId=${tenantId}`;
```

Comment:
Should also include a `top=1` query param since for a given form urn as recordId, it's possible to have multiple submissions and PDFs over time. When submission urn is the recordId, then only one file is expected.